apps/src/aiDifferentiation/notifications/AiDiffNotificationList.tsx

```diff
@@ -24,11 +24,16 @@ const AiDiffNotificationList: React.FC<AiDiffNotificationListProps> = ({
     HttpClient.fetchJson<AiDiffNotification[]>('/notifications', {}, undefined)
       .then(response => {
         setLoading(false);
-        const loadedNotifications = response?.value?.map(n => ({
-          ...n,
-          publishedAt: new Date(n.publishedAt),
-          readAt: n.readAt ? new Date(n.readAt) : null,
-        }));
+        const loadedNotifications =
+          response?.value
+            ?.map(n => ({
+              ...n,
+              publishedAt: new Date(n.publishedAt),
+              readAt: n.readAt ? new Date(n.readAt) : null,
+            }))
+            .sort(
```

Comment:
Now that we have contentful and rails backed notifications, the response is unsorted and we need to do it on the frontend. (note: video was taken before sorting was added)